Code/RDGeneral/types.cpp

```diff
@@ -46,6 +46,9 @@ const std::string _MolFileBondAttach = "_MolFileBondAttach";
 const std::string _MolFileBondType = "_MolFileBondType";
 const std::string _MolFileBondStereo = "_MolFileBondStereo";
 const std::string _MolFileBondCfg = "_MolFileBondCfg";
+const std::string _NonExplicit3DChirality =
```

Comment:
Given that this is only used in Chirality.cpp, I think it makes sense to define it there

Reply:
ok